src/actions/actions.ts

```diff
@@ -2154,7 +2151,7 @@ export class ShiftYankOperatorVisual extends BaseOperator {
 
 @RegisterAction
 export class DeleteOperatorXVisual extends BaseOperator {
```

Comment:
Decided not to change the name because <Del> is just a synonym in visual mode. Come to think of it, can't we combine `d` in visual mode with this as well?